lib/LocalEchoController.js

```diff
@@ -36,12 +36,19 @@ export default class LocalEchoController {
     this._cursor = 0;
     this._activePrompt = null;
     this._activeCharPrompt = null;
-    this._termSize = {
```

Comment:
As a good practice towards V8 optimisation it's a good idea to describe the object shape in the constructor and avoid mutating it later. So I would vote for keeping some defaults here, like so:
```js
    this._termSize = {
      cols: 0,
      rows: 0
    }
```